androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -113,6 +125,10 @@ fun EnhancedVideoPlayer(
         )
     }
 
+    fun setIsControlsVisible(value: Boolean) {
```

Comment:
nit: wrong english for `isControls` (plural). I'd rename to `setControlsVisibility(visible: Boolean)`